sale_commission/report/sale_commission_analysis_report_view.xml

```diff
@@ -103,8 +103,7 @@
     <record id="action_sale_commission_analysis_report" model="ir.actions.act_window">
         <field name="name">Commission Analysis</field>
         <field name="res_model">sale.commission.analysis.report</field>
-        <field name="view_type">form</field>
-        <field name="view_mode">pivot,graph</field>
+        <field name="binding_view_types">form,pivot,graph</field>
```

Comment:
I think the only thing you should have done here is delete `<field name="view_type">form</field>`